gavel/models/item.py

```diff
@@ -33,16 +33,21 @@ class Item(db.Model):
     sigma_sq = db.Column(db.Float)
 
 
-    def __init__(self, name, location, description):
+    def __init__(self, name, location, description, desired_prizes):
         self.name = name
         self.location = location
         self.description = description
         self.mu = crowd_bt.MU_PRIOR
         self.sigma_sq = crowd_bt.SIGMA_SQ_PRIOR
+        
+        tracks_set = set(desired_prizes)
```

Comment:
Instead of doing this here, can we do it in admin.py instead? So that once it gets to the constructor, it's already a set.